src/index.html

```diff
@@ -1,13 +1,46 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+      crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet"
+      href="./style.css">
+  </head>
+  <body>
+    <div class="card" data-qa="card"
+    >
+      <img
+        src="./images/imac.jpeg"
+        alt="foto"
+        class="foto_image"
+      >
+      <h2 class="title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+      <h3 class="product-code">Product code:195434</h3>
+      <div class="stars stars--4">
```

Comment:
Don't forget to add empty lines between multiline sibling blocks of HTML
```suggestion
      <h3 class="product-code">Product code: 195434</h3>
      
      <div class="stars stars--4">
```